diff -ur xarchiver-0.5.3.orig/src/window.c xarchiver-0.5.3/src/window.c
--- xarchiver-0.5.3.orig/src/window.c	2014-02-09 16:49:07.000000000 +0100
+++ xarchiver-0.5.3/src/window.c	2014-08-15 22:29:24.498161762 +0200
@@ -233,9 +233,9 @@
 {
 	int response;
 
-	dialog = gtk_message_dialog_new (window,mode,type,button,message1);
+	dialog = gtk_message_dialog_new (window,mode,type,button,"%s",message1);
 	gtk_dialog_set_default_response (GTK_DIALOG (dialog),GTK_RESPONSE_NO);
-	gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dialog),message2);
+	gtk_message_dialog_format_secondary_text (GTK_MESSAGE_DIALOG (dialog),"%s",message2);
 	response = gtk_dialog_run (GTK_DIALOG (dialog));
 	gtk_widget_destroy (GTK_WIDGET (dialog));
 	return response;
@@ -513,7 +513,7 @@
 			g_fprintf (stream,_("Comment:\n"));
 			if (bp)
 				g_fprintf(stream,"</b><pre>");
-			g_fprintf (stream,archive[idx]->comment->str);
+			g_fprintf (stream,"%s",archive[idx]->comment->str);
 			if (bp)
 				g_fprintf(stream,"</pre>");
 			g_fprintf (stream,"\n");
